src/tests/SelectedRevisionsTable.test.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+
+import SelectedRevisionsTable from '../components/Search/SelectedRevisionsTable/SelectedRevisionsTable';
+import { render, store, screen } from './utils/test-utils';
+
+describe('Search View', () => {
+  let mockStore: any;
+  let mockDispatch: any;
+
+  beforeEach(() => {
+    mockStore = jest.spyOn(store, 'getState');
+    mockDispatch = jest.spyOn(store, 'dispatch');
+  });
+
+  it('renders correctly when there are no revisions', () => {
+    mockStore.mockReturnValue({
+      search: {},
+      compareTable: {
+        revisions: [],
+      },
+    } as any);
+
+    render(<SelectedRevisionsTable />);
+    expect(screen.getByText('Add some revisions')).toBeInTheDocument();
+    expect(document.body).toMatchSnapshot();
+  });
+
+  it('should render correctly when there are revision and delete accordingly', async () => {
+    mockStore.mockReturnValue({
+      search: {},
+      compareTable: {
+        revisions: [
+          {
+            id: 0,
+            revision: 'coconut',
+            author: 'john@python.com',
+            push_timestamp: 42,
+            repository_id: 4,
+            revisions: [
+              {
+                revision: 'coconut',
+                author: 'john@python.com',
+                comments: 'This is my first message',
+              },
+            ],
+          },
+        ],
+      },
+    } as any);
```

Comment:
later on we can update this test to use data from the fixtures file (will be added in #90 )

Reply:
sure, would you wanna create a jira item for this?